{
    "CVE_data_meta": {
        "ASSIGNER": "secalert@redhat.com",
        "ID": "CVE-2017-7509",
        "STATE": "PUBLIC",
        "UPDATED": "2020-06-15"
    },
    "cna-container": {
        "affected": {
            "vendors": [
                {
                    "products": [
                        {
                            "versions": [
                                {
                                    "version_value": "pki-common-8.1.20-1"
                                }
                            ]
                        }
                    ],
                    "vendor_name": "Red Hat"
                }
            ]
        },
        "descriptions": [
            {
                "lang": "eng",
                "value": "An input validation error was found in Red Hat Certificate System's handling of client provided certificates before 8.1.20-1. If the certreq field is not present in a certificate an assertion error is triggered causing a denial of service."
            }
        ],
        "metrics": [
            {
                "cvss-v_3_0": {
                    "vectorString": "3.5/CVSS:3.0/AV:N/AC:L/PR:L/UI:R/S:U/C:N/I:N/A:L",
                    "version": "3.0"
                }
            }
        ],
        "problemtypes": {
            "descriptions": [
                [
                    {
                        "CWE-ID": "CWE-20",
                        "description": "CWE-20",
                        "lang": "eng",
                        "type": "CWE"
                    }
                ]
            ]
        },
        "references": [
            {
                "name": "RHSA-2017:2560",
                "refsource": "REDHAT",
                "url": "https://access.redhat.com/errata/RHSA-2017:2560"
            },
            {
                "name": "1039248",
                "refsource": "SECTRACK",
                "url": "http://www.securitytracker.com/id/1039248"
            },
            {
                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2017-7509",
                "refsource": "CONFIRM",
                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2017-7509"
            }
        ]
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "5.0"
}